config/initializers/geocoder.rb

```diff
@@ -37,3 +37,15 @@
 else
   Geocoder.configure(defaults)
 end
+
+module BingOverrideURLParams
+  def query_url_params(query)
+    { c: 'en-gb' }.merge(super)
+  end
+end
+
+module Geocoder::Lookup
+  class Bing < Geocoder::Lookup::Base
+    prepend BingOverrideURLParams
```

Comment:
This creates our own definition of `Geocoder::Lookup::Bing`, potentially preventing the original definition from loading.

I think instead this should be as follows

```ruby
Geocoder::Lookup::Bing.prepend(BingOverrideURLParams)
```

or if that isn't allowed

```ruby
Geocoder::Lookup::Bing.send(:prepend, BingOverrideURLParams)
```

or as an absolute fallback

```ruby
Geocoder::Lookup::Bing.class_eval do
  prepend BingOverrideURLParams
end
```